tests/e2e/bin/bootstrap-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Any reason not to pass these env variables in `docker-compose.yml` or `.env` file?

Reply:
That's basically copy-pasted file from https://github.com/WordPress/gutenberg/blob/master/bin/bootstrap-env.sh

I'm not sure about downsides of the current approach, as well as for other options. I mean it's just works 🤷‍♂ 

Happy to hear your opinion on that!

Reply:
You might not need the file if you add the env variables directly in composer file. I just think it's more readable to have things in one place. More in docs https://docs.docker.com/compose/environment-variables/

See _"Set environment variables in containers"_ bit.

Totally not a blocker for merging tho. :-)

Reply:
oh, I think these env variables are actually used to set-up a docker container via `./tests/e2e/bin/setup-travis-e2e-tests.sh` script, and not really used by the container itself. e.g. `DOCKER_COMPOSE_FILE_OPTIONS` which includes the path to `docker-compose` file

Reply:
Oh I see so `./tests/e2e/bin/setup-travis-e2e-tests.sh` is meant to be run locally, not inside any container. Gotcha :thumbsup: